list.go

```diff
@@ -86,7 +86,7 @@ func (l *ListCommand) Execute(args []string) error {
 }
 
 func (l *ListCommand) ListAsJSON(body []byte) error {
-	// newlineify the JSON for one marker per line
+	// newlinify the JSON for one marker per line
```

Comment:
Google found 4 pages w/ newlinify and 1 w/ newlineify.

I'm happy to drop this change...